sw/device/rom_exts/manifest.rs.tpl

```diff
@@ -0,0 +1,19 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+% for name, region in regions:
+/// Manifest field ${name} offset from the base.
+pub const ${region.offset_name().as_c_define()}:u32 = ${region.base_addr};
+
+/// Manifest field ${name} size in bytes and words.
+pub const ${region.size_bytes_name().as_c_define()}:u32 = ${region.size_bytes};
+pub const ${region.size_words_name().as_c_define()}:u32 = ${region.size_words};
```

Comment:
You still need a comment for *each* of the `pub const` definitions. @mcy wants to make it a warning to forget docs, which is why he's asked for this.

Reply:
Oh, my bad - yes, will fix them up and push!